phenopacketlab-io/src/main/java/org/monarchinitiative/phenopacketlab/io/PhenopacketResource.java

```diff
@@ -4,11 +4,11 @@
 
 import java.util.Objects;
 
-class PhenopacketResource implements Resource {
+public class PhenopacketResource implements Resource {
```

Comment:
Why do we need this to be public?

Reply:
I think I see - for the Jackson serializer. This is a pain in the bum. However, I think I found a way how to fix this without making the implementation public. I'll commit the code. Please resolve this to acknowledge.. :)